projects/freeimage/project.yaml

```diff
@@ -1,4 +1,5 @@
 homepage: "http://freeimage.sourceforge.net/"
+main_repo: 'https://svn.code.sf.net/p/freeimage/svn/'
```

Comment:
@oliverchang is it even useful for main_repo to be filled out for an svn repo? I think the only use is for building specific commits right?

Reply:
There are some other `svn` repos, e.g., [`projects/libteken/project.yaml`](https://github.com/google/oss-fuzz/blob/master/projects/libteken/project.yaml#L12).

Reply:
Yeah it's still useful to have. Other folks may use this to check if a project is being fuzzed on OSS-Fuzz